OpenRA.Mods.Common/Widgets/Logic/Editor/TilingPathToolLogic.cs

```diff
@@ -0,0 +1,122 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using OpenRA.Graphics;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Widgets;
+
+namespace OpenRA.Mods.Common.Widgets.Logic
+{
+	public class TilingPathToolLogic : ChromeLogic
```

Comment:
```suggestion
	[IncludeStaticFluentReferences(typeof(UpdateTilingPathPlanEditorAction), typeof(PaintTilingPathEditorAction))]
	public class TilingPathToolLogic : ChromeLogic
```

this will reveal that localizations are missing

Reply:
Ah, with this I can also move the label-tiling-path-tool definition back to the common ftl. (Assuming there's no problem with the fluent reference now being a const string.)